wzdx/experimental_combination/attenuator.py

```diff
@@ -0,0 +1,285 @@
+import json
+from ..tools import (
+    cdot_geospatial_api,
+    geospatial_tools,
+    combination,
+    date_tools,
+    wzdx_translator,
+)
+import logging
+from datetime import datetime, timedelta
+
+ATTENUATOR_TIME_AHEAD_SECONDS = 30 * 60
+ISO_8601_FORMAT_STRING = "%Y-%m-%dT%H:%M:%SZ"
+
+
+def main(outputPath="./tests/data/output/wzdx_attenuator_combined.json"):
+    with open(
+        "./wzdx/sample_files/raw/geotab_avl/attenuator_combination_geotab.json"
```

Comment:
Are these files replaced on the fly? Or are we hard coding to a single example instance since this is experimental?